packages/components/src/DocumentViewer/components/PanViewer.tsx

```diff
@@ -81,74 +51,24 @@ export default class PanViewer extends React.Component<IProps> {
     rotation: 0
   }
 
-  renderPanZoomControls = (centerController?: boolean) => {
-    return (
-      <ControlsContainer centerController={centerController}>
-        <div onClick={this.zoomIn}>
-          <ZoomIn />
-        </div>
-        <div onClick={this.zoomOut}>
-          <ZoomOut />
-        </div>
-        <div onClick={this.rotateLeft}>
-          <RotateLeft />
-        </div>
-      </ControlsContainer>
-    )
-  }
   render() {
```

Comment:
Guess we could functionalize this component aswell as it's a simple one

Reply:
this has been applied